Makefile

```diff
@@ -180,9 +180,12 @@ endif
 ginkgoBuild: generate
 	make -C ./test/e2e ginkgo-build
 
-test:
+test: test-node-bootstrapper
```

Comment:
I think eventually we should follow up for testing every child go mod w.r.t the repo root